relay-config/src/config.rs

```diff
@@ -1032,10 +1035,29 @@ impl Default for Processing {
             attachment_chunk_size: default_chunk_size(),
             projectconfig_cache_prefix: default_projectconfig_cache_prefix(),
             max_rate_limit: default_max_rate_limit(),
+            normalize_events: NormalizeEvents::Default,
         }
     }
 }
 
+/// Configuration for the level of normalization this Relay should do.
+#[derive(Clone, Debug, Default, Serialize, Deserialize, Eq, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum NormalizeEvents {
+    /// Disables normalization for events coming from internal relays.
```

Comment:
etc.

```suggestion
    /// Disables normalization for events coming from internal Relays.
```

Reply:
Done in [d936ff4](https://github.com/getsentry/relay/pull/3377/commits/d936ff4d29e3814fdc916f42dd955d03cbb8dfa1).